### ISSUE TITLE
Copy path of a selected node

### ISSUE THREAD

**Issue by qkxie** [triggerdotdev/jsonhero-web#165]
Hope there is a feature to copy the path of the selected in the format of a target language.

for example:

```
{"a": {"b": {"c": 1}}}
```

In Python, the copied value should be: xxx['a']['b']['c']

In JavaScript, the copied value should be: xxx.a.b.c

**Comment by changdy** [comment]
It is a good idea . and very useful

**Comment by Maximization** [comment]
Yes! This is exactly what I'm looking for. My use case is to paste a deeply nested object in JSON Hero, navigate to the value I want through the UI, and would love to copy with a single click the JavaScript I need to use to get that value programatically.

**Comment by qkxie** [comment]
I posted this issue 3 months ago, but the author hasn't even replied to me, let alone started developing this feature.

**Comment by CrudeCreations** [comment]
> I posted this issue 3 months ago, but the author hasn't even replied to me, let alone started developing this feature.

These guys also have a full time job, the code's open source and if you really need a feature you can write it yourself and make a PR. [Here I threw one togethere for you](https://github.com/triggerdotdev/jsonhero-web/pull/181#issue-2229840555).

I've added a 'Copy Path' button that lets you copy in JS or Python at the moment. Use my fork until it's they add this feature if you're desperate.
